File: nivel 1/fase 1/2019/nota.c
#include <stdio.h>

int main() {
    int b, t;
    scanf("%d", &b);
    scanf("%d", &t);

    int trapezio = ((b + t) * 160) / 2;
    int valor = (160 * 160) / 2;

    if (trapezio == valor) {
        printf("0\n");
    } else if (trapezio > valor) {
        printf("1\n");
    } else {
        printf("2\n");
    }

    return 0;
}

File: nivel 1/fase 1/2019/nota.py
b = int(input())
t = int(input())

trapezio = ((b + t) * 160)/2

valor = (160 * 160)/2

if trapezio == valor:
    print('0')
elif trapezio > valor:
    print('1')
else:
    print('2')

# https://neps.academy/br/exercise/465